src/parser.rs

```diff
@@ -92,13 +88,154 @@ impl Uuid {
     /// [`parse_str`]: #method.parse_str
     #[inline]
     pub const fn try_parse(input: &str) -> Result<Uuid, Error> {
-        match imp::try_parse(input) {
+        match try_parse(input) {
             Ok(bytes) => Ok(Uuid::from_bytes(bytes)),
             Err(_) => Err(Error(ErrorKind::Other)),
         }
     }
 }
 
+#[inline]
```

Comment:
I am not sure about this inline...

while in a `const` context this works fine... however in non-`const` this is bound to create a lot of bloat

Reply:
These inlines don't seem to be doing much for us, so I removed them :+1:

Reply:
These inlines have observable effects on the performance.

If you want to avoid code bloat, just write a wrapper function with `#[inline(never)]`

https://rust-lang.github.io/rust-clippy/master/index.html#missing_inline_in_public_items

Reply:
This is an interesting point. 

@KodrAus Thoughts?

Reply:
@Nugine I didn’t observe any difference in removing these two specifically, even at the level of optimization we’re at now where small perturbations make a big difference, so opted for fewer inlines.